packages/visx-demo/src/sandboxes/visx-geo-albers-usa/Example.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { AlbersUsa } from '@visx/geo';
+import * as topojson from 'topojson-client';
+import topology from './usa-topo.json';
+import stateAbbrs from './us-abbr.json';
+import { geoCentroid } from 'd3-geo';
+
+export const background = '#EBF4F3';
+
+export type GeoAlbersUsaProps = {
+  width: number;
+  height: number;
+  fullSize: boolean;
+};
+
+interface FeatureShape {
+  type: 'Feature';
+  id: string;
+  geometry: { coordinates: [number, number][][]; type: 'Polygon' };
+  properties: { name: string };
+}
+
+// @ts-ignore
+const { features: unitedStates } = topojson.feature(topology, topology.objects.states) as {
+  type: 'FeatureCollection';
+  features: FeatureShape[];
+};
+
+export const colors: Array<string> = ['#744DCA', '#3D009C', '#9020FF', '#C630FD']
```

Comment:
nit – just more consistent with syntax we use elsewhere

```suggestion
export const colors: string[] = ['#744DCA', '#3D009C', '#9020FF', '#C630FD']
```